sdk/js/src/api/stream/stream-node.js

```diff
@@ -85,11 +85,11 @@ export default async (payload, url) => {
       })
       stream.on('end', () => {
         notify(listeners[EVENTS.CLOSE])
-        listeners = null
+        listeners = []
```

Comment:
```suggestion
        listeners = {}
```

It seems weird to me to use `[]`, since `listeners` is usually a plain object and not an array instance.